src/ResourceManager/Compute/Commands.Compute/help/Get-AzureRmGallery.md

```diff
@@ -0,0 +1,115 @@
+---
+external help file: Microsoft.Azure.Commands.Compute.dll-Help.xml
+Module Name: AzureRM.Compute
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.compute/get-azurermgallery
+schema: 2.0.0
+---
+
+# Get-AzureRmGallery
+
+## SYNOPSIS
+Get or list galleries.
+
+## SYNTAX
+
+### DefaultParameter (Default)
+```
+Get-AzureRmGallery [[-ResourceGroupName] <String>] [[-GalleryName] <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceIdParameter
+```
+Get-AzureRmGallery [-ResourceId] <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get or list galleries.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzureRmGallery -ResourceGroupName $rgname -GalleryName $galleryName
```

Comment:
@hyonholee for the examples in the markdown files you're adding, would you mind also showing the output that is returned from the command execution? This will give users a better idea of what the object looks like that they get back